src/plugins/recordTypes/place/optionLists.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  ucbgPlaceTermTypes: {
+    values: [
+      'Synonym',
+      'Verified Geographic Name',
+      'Reverse Qualified Name',
+      'Short Name',
+      'Unverified Geographic Name',
+    ],
+  },
+};
```

Comment:
Need to customize the placeHistoricalStatuses option list. See botgarden-authority-place-termList.xml in the app layer. I think it's just adding the unknown value.